taskana-adapter-tasklistener-example/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we really need persistence.xml ? After all, we are not using a persistence manager ....